apps/esbuild-example/public/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+  <head>
+    <title>@stylexjs/esbuild-plugin</title>
+    <meta charset="utf-8" />
+    <style>
+      body {
+        box-sizing: border-box;
+        padding: 0;
+        margin: 0;
+      }
```

Comment:
Since you have `useCSSLayers: true`, the reset needs to be in a layer as well.

```suggestion
      @layer reset {
        html {
          box-sizing: border-box;
          padding: 0;
          margin: 0;
        }
      }
```